docs/.vuepress/public/presets/toml/one-dark.toml

```
@@ -0,0 +1,304 @@
+"$schema" = 'https://starship.rs/config-schema.json'
+
+# Configures the format of the prompt.
+format = """
+[](color_green)\
+$os\
+$username\
+[](bg:color_orange fg:color_green)\
+$directory\
+[](fg:color_orange bg:color_cyan)\
+$git_branch\
+$git_status\
+[](fg:color_cyan bg:color_blue)\
+$c\
+$dotnet\
+$rust\
+$golang\
+$nodejs\
+$php\
+$java\
+$kotlin\
+$haskell\
+$python\
+$elixir\
+$elm\
+$gradle\
+$julia\
+$nim\
+$scala\
+[](fg:color_blue bg:mono3)\
+$docker_context\
+[ ](mono3)\
+$fill\
+[ ](mono4)\
+$time\
+$battery\
+[ ](mono4)\
```

To be more customizable, I would suggest putting the block in a conditional:
```suggestion
([ ](mono4)\
$time\
$battery\
[ ](mono4))\
```
That enables the user to disable the `[time]` and the `[battery]` module, without seeing a gray dot / pill.